libraries/freertos_plus/aws/greengrass/test/aws_test_ggd_system.c

```diff
@@ -166,6 +206,15 @@ TEST( GGD_System, GetGGCIPandCertificate )
                                                   cBuffer, /*lint !e971 can use char without signed/unsigned. */
                                                   ulBufferSize,
                                                   &xHostAddressData );
+            RETRY_EXPONENTIAL( xStatus - GGD_GetGGCIPandCertificate( clientcredentialMQTT_BROKER_ENDPOINT,
```

Comment:
`xStatus - `? Should it be `=`? Also, the previous call to `GGD_GetGGCIPandCertificate` was not deleted so this will run twice.

Reply:
Must have had too much caffeine.